### ISSUE TITLE
akita-ng-router-store Invalid route params

### ISSUE THREAD

**Issue by xperiments** [datorama/akita#82]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[X] Other... Please describe:
</code></pre>

## Current behavior

In a component, I am trying to get the actual route provided params but getting an empty params response.

With this route `test/:someParam/:other` and this url `test/100/200`, I can't access the `someParam` & `other values`

```
this._routerQuery.select(state => state.state.root.params).subscribe(params => { 
  // params is empty object here {}
})
```

## Expected behavior

For this route 
`test/:someParam/:other`

```
this._routerQuery.select(state => state.state.root.params).subscribe(params => { 
  params.someParam; <-- accesible
  params.other; <-- accesible
})
```

## What is the motivation / use case for changing the behavior?

What I expect is to get the actual route params.

I have done a simple patch in the RouterService to test if it works the expected way with this little change:

````
RouterService.prototype['serializeRoute'] = function(routerState) {

  let state: ActivatedRouteSnapshot = routerState.root;
  while (state.firstChild) {
    state = state.firstChild;
  }

  const { params } = state;

  return {
    url: routerState.url,
    params: params,
    queryParams: routerState.queryParams,
    fragment: routerState.fragment,
    data: routerState.data,
    paramMap: routerState.paramMap,
    queryParamMap: routerState.queryParamMap
  };
};
````

Also will be great to have the public akita-ng-router-store repo avaliable to issue errors there.

## Environment

<pre><code>
Angular version: 6.0.3

Browser:
- [X] Chrome (desktop) version 69
 
For Tooling issues:
- Node version: 8.x
- Platform:  OSX

**Comment by NetanelBasal** [comment]
Thanks! I will test this change and publish soon. ( also the lib )

**Comment by NetanelBasal** [comment]
Fixed in 1.0.0. Thanks.

I know that you wanted to make this PR so I added you as a contributor anyway :)

**Comment by xperiments** [comment]
Perfect!! Works in 1.0.0 for me.
Waiting to see the repo online.
I will be playing with this the weekend XDD
